app/controllers/investigations/ownership_controller.rb

```diff
@@ -69,4 +72,12 @@ def session_params
   def form
     @form ||= ChangeCaseOwnerForm.new(form_params)
   end
+
+  def get_potential_assignees
+    @selectable_users = [@investigation.owner_user&.object, current_user].uniq.compact
+    @team_members = User.get_team_members(user: current_user)
```

Comment:
I know this is an existing method, but I'd be tempted to refactor it to this, and remove the class method from `User`.

```suggestion
    @team_members = current_user.team.users.active
```